src/actions/marketo/marketo_transaction.ts

```diff
@@ -0,0 +1,236 @@
+/* tslint:disable no-console */
+import * as Hub from "../../hub"
+import { Queue } from "./queue"
+
+const MARKETO: any = require("node-marketo-rest")
+
+const numLeadsAllowedPerCall = 100
+
+// function log(...args: any[]) {
```

Comment:
Would just consider taking these comments out or changing to winston for consistency across the repo. @wilg in case he disagrees but I would bet we are on the same page there